openbas-front/package.json

```diff
@@ -36,7 +36,6 @@
     "html-react-parser": "5.2.5",
     "html-to-image": "patch:html-to-image@npm%3A1.11.13#~/.yarn/patches/html-to-image-npm-1.11.13-86e990604f.patch",
     "http-proxy-middleware": "3.0.5",
-    "i18n-auto-translation": "2.2.1",
```

Comment:
Are we sure we want to remove this ? @johanah29 ?

Reply:
@RomuDeuxfois It just got moved, it's still there with the same version